### ISSUE TITLE
Color selector and icon feels blury

### ISSUE THREAD

**Issue by shirok1** [flameshot-org/flameshot#2426]
### Feature Description

![image](https://user-images.githubusercontent.com/12044683/154020400-71ace36c-05dc-4f32-8b02-1c2a6a2b6d0d.png)

![image](https://user-images.githubusercontent.com/12044683/154020468-ee8baf4a-71a3-459a-a42a-ddba39500198.png)

Probably because of my desktop environment: KDE Plasma 5.23 on Xorg, with 150% "Global scale".

Resources in this repo seem to be vector image. If I may ask, how are they rendered? Are they first rendered into a bitmap and scaled(stretched) by Qt?

**Comment by shirok1** [comment]
I changed the "Global scale" to 100% and the icons become pixel perfect.
<img width="450" alt="image" src="https://user-images.githubusercontent.com/12044683/154022166-732b3285-f4fa-4158-b85a-99448c2b3e02.png">
<img width="233" alt="image" src="https://user-images.githubusercontent.com/12044683/154022325-f9ff6fab-321f-4cd0-a6fe-37187b2cf255.png">

**Comment by mmahmoudian** [comment]
@borgmanJeremy can this be because of the svg optimisation pull request?

**Comment by borgmanJeremy** [comment]
@mmahmoudian I dont think so, this is very strange. I think this is an issue in Qt itself, we don't do anything to render the svg other than provide the path to the svg.
